### ISSUE TITLE
Notification/comment on test command success and failure

### ISSUE THREAD

**Issue by geekodour** [prometheus/test-infra#252]
**With the current Prow setup:**
1.`/benchmark` : A comment is posted before `make deploy` is run, this comment contains all information about the test. If something fails while `make deploy` is running prow posts a comment about the prowjob exiting with an error code.
2. `/benchmark cancel` : No comment is posted

**With the new Github Actions setup:**

Current comments:
1.`/benchmark` : A comment is posted before `make deploy` is run, this comment contains all information about the test.
2. `/benchmark cancel` : A comment is posted before `make clean` is run. ( @krasi-georgiev recommends getting rid of this comment as we'll get a notification after 3 days anyway if this failed by the alertmanager)

ref https://github.com/prometheus/prombench/pull/248#issuecomment-520129011

I'd like to suggest a few more places for comments that  can make working with prombench from GitHub easier, this assumes the use of the StatefulSet implementation to run tests:

**Github Actions with StatefulSet**
1. `/benchmark`:  A comment is posted before make deploy is run, this comment contains all information about the test. Additionally, should contain the link to the deployment log for the statefulset in loki. (WIP)
change:
```
make deploy && echo 'deploy succeeded, now sleeping' && while sleep 86400; do :; done
```
to
```
if make deploy; then echo 'deploy succeeded, now sleeping'; while sleep 86400; do :; done; else make comment_deploy_failure; fi`
```
`comment_deploy_failure` is a Makefile target, which will call a script from `/scripts` directory which will post a comment to GitHub PR about that there was some failure in running `make deploy`

2. `/benchmark cancel` : No comment is posted

Some of these comments might be not necessary please provide you feedback and suggestions

**Comment by krasi-georgiev** [comment]
> 2. /benchmark cancel : A comment is posted before make clean is run. ( @krasi-georgiev recommends getting rid of this comment as we'll get a notification after 3 days anyway if this failed by the alertmanager)

afer some discussion we will keep the comment to give feedvack that the cancellation has starter. This will remove a confusion like: "Did I type the comment properly", "Did the system detect the webhook" etc.


> if make deploy; then echo 'deploy succeeded, now sleeping'; while sleep 86400; do :; done; else make comment_deploy_failure; fi`


lets wait for github actions a bit more as they should provide some form of feedback when a workflow failed. Maybe even if there is a feedback form can suggest this feature. All CI systems have something like this.

**Comment by nikita-mk** [comment]
https://help.github.com/en/articles/about-github-actions#notifications-for-workflow-runs 
Its there in the docs to send enable notifications for gh actions. Even an option to send notifications for failed workflows only.

To enable https://help.github.com/en/articles/choosing-the-delivery-method-for-your-notifications#choosing-the-delivery-method-for-your-notifications-about-github-actions

**Comment by geekodour** [comment]
Github Action issue to track commenting on failure:
https://github.community/t5/How-to-use-Git-and-GitHub/Github-Actions-Contexts-and-steps-if-conditionals-are-unusable/m-p/29669/highlight/true#M8294

**Comment by geekodour** [comment]
closed by #287
